pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -189,7 +191,7 @@ type ScrapeConfigSpec struct {
 	DigitalOceanSDConfigs []DigitalOceanSDConfig `json:"digitalOceanSDConfigs,omitempty"`
 	// KumaSDConfigs defines a list of Kuma service discovery configurations.
 	// +optional
-	KumaSDConfigs []KumaSDConfig `json:"kumaSDConfigs,omitempty"`
+	KumaSDConfigs KumaSDConfigs `json:"kumaSDConfigs,omitempty"`
```

Comment:
Hi @dev-priyanshu15 might I ask why this change? seems a bit unrelated to the change in the ScalewaySDConfig kubebuilder.